platform/darwin/scripts/style-spec-cocoa-conventions-v8.json

```diff
@@ -0,0 +1,9 @@
+{
+  "layout_symbol": {
+    "icon-image": "icon-image-name"
+  },
+  "paint_raster": {
+    "raster-brightness-min": "raster-brightness-minimum",
```

Comment:
minimumRasterBrightness reads better. We might as well make that change as long as we're renaming this property.